libkbfs/tlf_journal_test.go

```diff
@@ -1030,6 +1030,12 @@ func testTLFJournalResolveBranch(t *testing.T, ver MetadataVer) {
 	require.NoError(t, err)
 	require.False(t, flushed)
 
+	// The background worker was already paused, so we won't get a
```

Comment:
Later checks in the test depend on being able to manually flush things.  I don't want to shake up the test too much for this change, so leaving as is for now.